### PR TITLE
Zero app experience for getting started page

### PR DESCRIPTION
**Pull request by sreevatsanraman**

### CODE REVIEW

File: cdap-examples/CubeService/README.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# CubeService example
```
Comment (gokulavasan) [inline cdap-examples/CubeService/README.md]: Do we still use .md files?

File: cdap-examples/resources/weblog-adapter-config.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{ "template": "ETLBatch", "description": "StreamConversion ETL", "config": { "schedule": "*/1 * * * *", "source": { "name": "Stream", "properties": { "name": "logEventStream", "format" :"clf", "duration": "1h" } }, "sink": { "name": "Cube", "properties": { "name": "logEventCube", "dataset.cube.resolutions": "1", "dataset.cube.aggregation.agg1.dimensions": "remote_host","dataset.cube.aggregation.agg2.dimensions": "status",  "mapping.config": "{\"timestamp\": {\"sourceField\": \"ts\"}, \"tags\": [], \"dimensions\": [{\"name\":\"remote_host\", \"sourceField\":\"remote_host\"}, {\"name\":\"status\",\"sourceField\":\"status\"}],\"measurements\": [{\"name\": \"count\", \"type\": \"COUNTER\", \"value\": \"1\"}]}" } }, "transforms": [] } }
```
Comment (gokulavasan) [inline cdap-examples/resources/weblog-adapter-config.json]: this should be formatted and should not be in one line.

also 1minute schedule is too aggressive for MR programs.

File: cdap-examples/CubeService/README.md (hunk 1 above)
Comment (sreevatsanraman) [inline cdap-examples/CubeService/README.md]: yes. we can change it later. other examples use .md as well

File: cdap-examples/resources/weblog-adapter-config.json (hunk 2 above)
Comment (sreevatsanraman) [inline cdap-examples/resources/weblog-adapter-config.json]: We don't want the users to wait longer to see results. I have formatted json.

**Comment by gsps1** [comment]
LGTM

**Pull request merged**